pkg/db/db.go

```diff
@@ -134,6 +136,13 @@ func extractTransaction(ctx context.Context) *sqlx.Tx {
 	}
 }
 
+func removeTransactionFromContext(ctx context.Context) context.Context {
```

Comment:
why this is needed?

Reply:
After a transaction already commited, the transaction detail is still stored in context. So, when storing log, it return error since it still reading and use the transaction.

Reply:
Then lets make this public and call it everytime we need to clean up transaction. Lets keep rollback and commit contract remain uniform across the repository

Reply:
I was thinking that making it public would make a chance of removing transaction halfway without commiting or rollback, but sure.

Reply:
It should be similar like context.WithoutCancel but this one is WithoutTx

Or we could just create a new context when logging activity. But we lost all previous information in context in case we need it.

Reply:
oh 😅 I believed I have tried creating new context while implementing log back then and it won't work. Tried it again this morning and it works, so will use that approach.